standards/request-response.md

```diff
@@ -493,6 +493,36 @@ The usage of non-standard headers is not considered custom headers. For example,
 SPS-CORS-Error: bad origin
 ```
 
+#### SPS-Execution-Context
+
+**Type**: Request
+
+**Support**: OPTIONAL
+
+**Description**: The Execution Context header provides a standard method of indicating the dataflow context that a request should be processed with. This is used to differentiate or test changes in customer configuration of non-production data. This isolates the version of the configuration state used in a request and allows identifying transactions that should leverage a specific tagged or aliased configuration set.
+
+- A request without the presence of this header **MUST** indicate that the execution context is under the `production` dataflow and customer configuration, if applicable.
+- The header value contains any dynamic __string__ value representing a named configuration or tagged dataflow preset.
+    - __preprod__ - Well-known value used to represent non-production customer configuration and can be supported as a static mode in some legacy services without full dynamic support. Usage of dynamically named configuration sets is preferred.
+- An invalid or unsupported header value provided **MUST** result in a `400 - Bad Request` following standard [error format](errors.md#400-bad-request).
+- The header value **MUST NOT** exceed a maximum length of 100 characters.
+- The header value **SHOULD** contain human-readable tag for the context.
+- The header **MUST NOT** be used in a response.
+- The header **SHOULD** be propagated to any outgoing requests to retain the context for downstream usage.
```

Comment:
Do we need to add language regarding services that support both `preprod` and `prod` in the same context?  An example would be DEX supporting the `preprod` context which depends on Rules management that makes no distinction between `preprod` and `prod`.  Obviously we wouldn't expect Rules Management to return a 400.

Reply:
Or should we leave it up to the individual API's documentation to make it clear there is no distinction.

Reply:
If I understand correctly, this issue is because your jumping ahead to "phase 2" of this transition where you wouldn't use "preprod", it uses dynamic rules and overrides. Take a look at internal docs about preprod context to see if that makes sense...